src/Core/System/SalesChannel/Event/SwitchContextEvent.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Shopware\Core\System\SalesChannel\Event;
+
+use Shopware\Core\Framework\Validation\DataBag\RequestDataBag;
+use Shopware\Core\Framework\Validation\DataValidationDefinition;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+
+class SwitchContextEvent
```

Comment:
please implement the ShopwareEventInterface and getters like in other events for now. see comment above